src/python/pants/backend/python/tasks2/pytest_run.py

```diff
@@ -568,6 +569,7 @@ def parse_error_handler(parse_error):
 
   def _pex_run(self, pex, workunit_name, args, env):
     with self.context.new_workunit(name=workunit_name,
+                                   cmd=pex.cmdline(args),
```

Comment:
It does, but we delegate everything except adding the PEX_PATH prefix to the pex library, so it would have to compute 2 different answers. To fix I'd need to introduce something like this to pex and consume from pants: https://github.com/pantsbuild/pants/blob/master/src/python/pants/java/executor.py#L26-L89